README.md

```diff
@@ -58,6 +58,33 @@ the following environment variables to your container:
   - `DB_USER` (default: `root`)
   - `DB_PASS`
 
+### Postgres
+
+In addition to the linked MySQL backend, you can alternatively use a Postgres
+container. The configuration is very similar to that of MySQL as seen in the
+`docker-compose.yml` file, with a couple small changes.
+
+In the `web` section, you'll need to link in a container under the name
```

Comment:
as Pythonasta i say explicit is better than implicit, why not add a full (yet minimal) working example?